packages/commonwealth/server/controllers/server_communities_methods/update_community.ts

```diff
@@ -71,6 +78,9 @@ export async function __updateCommunity(
     if (!user.isAdmin && !userMembership) {
       throw new AppError(Errors.NotAdmin);
     }
+    ownerOfChain = addresses.filter(
+      (a) => a.community_id === chain.id && a.role === 'admin',
+    )[0];
```

Comment:
This will throw an error if the requester does not have any admin addresses.

Reply:
nvm. I see the issue here (Will throw NPE)